### DISCUSSION TITLE
Active

### DISCUSSION THREAD

**Discussion by davewatson91** [DataForNerds/public#684] [General]
Hi @ChrisKibble! 

Just stumbled across your project here, and it seems amazing for automating some of hte tasks I'm looking at (especially in the wake of the Outlook CVE). 

Are you planning on updating the main branch at all? I can see that there is a couple of other branches, but not entirely sure how to get their JSON files (not sure how the github actions work for branches / web access). 

Thanks! 
Dave

**Comment by ChrisKibble** [comment]
Hey @davewatson91 - thanks for checking in! The automation kicks the JSON into the branches for me to check on and then I make sure stuff didn't get too crazy before I accept them. I've been a little behind :).   Just accepted them all though so it should be slightly more up to date, although Microsoft has started changing their sources so some code changes are likely coming.

**Reply by davewatson91** [reply]
No worries :) I actually just pulled the guts of your script out, and have set it up to run alongisde each machine I run it on - it generate more traffic, but is a fairly low overhead in reality.. 

Do appreciate your work!
